.travis.yml

```diff
@@ -1,24 +1,18 @@
-sudo: required
-dist: trusty
-# Force travis to use its minimal image with default Python settings
+sudo: required 
+dist: trusty 
 language: generic
-compiler:
-  - gcc
-
+notifications:
```

Comment:
I strongly discourage you to set `on_failure: change`. If travis fails this should be a permanent thread to your mailbox until it is fixed properly :-)